control/cli.py

```diff
@@ -36,11 +44,40 @@ def __init__(self):
             prog="python3 -m control.cli",
             description="CLI to manage NVMe gateways")
         self.parser.add_argument(
-            "-c",
-            "--config",
-            default="ceph-nvmeof.conf",
+            "--server-address",
+            default="ceph-nvmeof_ceph-nvmeof_1",
+            type=str,
+            help="Server address",
+        )
+        self.parser.add_argument(
+            "--server-port",
+            default=5500,
+            type=int,
```

Comment:
`argparse` provides [`FileType`](https://docs.python.org/3/library/argparse.html#filetype-objects) objects to ease parsing & validation of file-like arguments.